src/context/ModalProvider/ModalContainer.tsx

```diff
@@ -221,6 +221,14 @@ const WalletDrawer = makeSuspenseful(
   ),
 )
 
+const AddAddressModal = makeSuspenseful(
+  lazy(() =>
+    import('@/components/Modals/Send/AddAddressModal').then(({ AddAddressModal }) => ({
```

Comment:
suggestion: should probably use the address book vernacular here - intent not super clear from component/module name.
Perhaps even "save" terminology?